platforms/pf-power-supply/Makefile.dep

```diff
@@ -1,10 +1,6 @@
 # RIOT modules
 USEMODULE += event
-USEMODULE += periph_gpio_irq
-USEMODULE += periph_qdec
-USEMODULE += motor_driver
 USEMODULE += printf_float
-USEMODULE += ps
 USEMODULE += scanf_float
```

Comment:
I think you failed your commit squash :stuck_out_tongue:

Reply:
yes ^^ Fixed.